### DISCUSSION TITLE
Howto setup with Application Service?

### DISCUSSION THREAD

**Discussion by wrenix** [matrix-org/matrix-authentication-service#2490] [Q&A]
Current my mautrix does not startup with the old synapsr server.
The whois failed

PS: I know, that the Encryption does not work yet:
https://github.com/matrix-org/matrix-authentication-service/commit/c0afe98507dc839970c3d3761614668f56efbc08

**Comment by sandhose** [comment]
On that front, if you have encryption disabled, it should work like before. Can you make sure the application service is registered in the Synapse configuration file like before?

What error is mautrix giving on startup?

**Comment by wrenix** [comment]
I solve my Problem -> the bridge muss also pass the matrix-authentification-services 

(and not communicate the synapse direct on internal way)
